data_transfer/schemas/record.py

```diff
@@ -32,10 +32,10 @@ class Record(BaseModel):
     dmp_folder: Optional[str]
 
     # Each stage of the pipeline
-    is_downloaded: Optional[bool] = False
-    is_processed: Optional[bool] = False
-    is_prepared: Optional[bool] = False
-    is_uploaded: Optional[bool] = False
+    is_downloaded: bool = False
+    is_processed: bool = False
+    is_prepared: bool = False
+    is_uploaded: bool = False
```

Comment:
I didn't understand why these were optional (i.e. could be assigned `None`) if we default them to `False` anyway..